### PR TITLE
Feature/generate test data for maintenance script

### PR DESCRIPTION
**Pull request by kvahed**
fixes arangodb/team-clifton-planning#28

### CODE REVIEW

File: scripts/generateMaintenanceTestData.sh [hunk 1]
```diff
@@ -1,22 +1,61 @@
...
+
+
 
 header="R\"=("
 footer=")=\""
 
 outfile=Plan.json
 echo $header > $outfile
-curl -s localhost:4001/_api/agency/read -d'[["/arango/Plan"]]'|jq .[0].arango.Plan >> $outfile
+curl [::]:4001/_api/agency/read -sLd'[["/arango/Plan"]]'|jq .[0].arango.Plan >> $outfile
```
Comment (matthewvon) [inline scripts/generateMaintenanceTestData.sh]: needs to be localhost for ipv4 compatibility

File: scripts/generateMaintenanceTestData.sh [hunk 2]
```diff
@@ -1,22 +1,61 @@
 #!/bin/bash
-# FIXMEMAINTENANCE: please add a couple of lines about why this script exists
+#
+# Generate test data for maintenance unit tests
+#
+
+# collections for system database
+echo "== Creating collection _system/bar =="
+curl [::]:8530/_api/collection -sd \
```
Comment (matthewvon) [inline scripts/generateMaintenanceTestData.sh]: default endpoint is 8529, is that the intention?

**Comment by matthewvon** [comment]
My first attempts with this script failed.  I will try again later.  It seems to assume a configuration I do not have.

File: scripts/generateMaintenanceTestData.sh (hunk 2 above)
Comment (kvahed) [reply scripts/generateMaintenanceTestData.sh]: just leave as is. i'll fix for next round

File: scripts/generateMaintenanceTestData.sh (hunk 1 above)
Comment (kvahed) [reply scripts/generateMaintenanceTestData.sh]: we don't need IP4, really!
let freaking ip 4 die. we don't need it. we have tons o scripts, that don't care about that. come on folks, let's not do this kind of pseudo fixes. this is an internal script like startLocalCluster.

**Pull request merged**